lib/lutaml/model/serialize.rb

```diff
@@ -21,21 +21,21 @@ module Serialize
 
       def self.included(base)
         base.extend(ClassMethods)
+        base.initialize_attrs(base)
```

Comment:
I think this will cause an issue in multilevel inclusion, i.e 

```ruby
class One
  include Lutaml::Model::Serialize
end

class Two
  include One
end
```

@0arm0 Can you add a test case for this as well?